script/Ciel/Day 5/05_01_CIEL05_3E.txt

```diff
@@ -251,82 +262,91 @@ Inside is―――
+She's fighting against that vampire all on her own.
 }
 [sha:9b001ac172889581d51d0a0c192c763fd15aa719]{
 -- Page 593, Offset 12407.
 -- マイナス100℃に迫る極寒。
-Extreme cold approaching -100℃.
+The intense cold approaches -100℃.
 }
 [sha:4e459c85d7fe6985da0efb02c3760bd39e6e3648]{
 -- Page 593, Offset 12408.
 -- この惑星の地表では起こりえない超低温。
-An extraordinarily low temperature that has never been reached on this planet's surface.
+// Added 'naturally' occur because we have made things _way_ colder than that on purpose
+A cryogenic temperature that could never occur naturally on the surface of this planet.
 }
 [sha:f337eaec63f7fa47b79a3ecdc25dd70c46aced09]{
 -- Page 593, Offset 12409.
 -- あらゆる動植物の生存を許さぬ冷気の中、二つの騎影が衝突する。
-In the midst of a cold that precludes the survival of all plants and animals, two knightly figures are clashing.
+Within that cold so intense that no plant or animal could survive,
+ two knightly figures clash with one another.
 }
 [sha:f1300f98c234c9ff59718654e4a8a63e94b7bc95]{
 -- Page 594, Offset 12410.
 -- 一騎は呪いの源、ヴローヴ・アルハンゲリ。
-One knight is the source of this curse, Vlov Arkhangel.
+The first knight is the source of this curse, Vlov Arkhangel.
 }
 [sha:ef472ed8e42fd712cb13f36154915c396ea2e69a]{
 -- Page 594, Offset 12411.
 -- 一騎は<拷問具|アーマー>に身を包んだ、少女の姿をした代行者。
-The other is a young female Executor clad in <armor|a n  i n s t r u m e n t  o f  t o r t u r e     >.
+// Armor padded to a phrase so the chungus ruby doesn't eclipse it so badly
+The other is a young female Executor clad in <a suit of armor|a n  i n s t r u m e n t  o f  t o r t u r e     >.
 }
 [sha:a64320d127cf3e1cbf3621c3fe403e89600eafe1]{
 -- Page 595, Offset 12412.
 -- 吹雪によって閉ざされた視界。
-My whole field of vision is blocked by the blizzard.
+Their field of vision is hemmed in by the blizzard.
 }
 [sha:2c7d44631d11ca091845c7774de6a6520e93cfe7]{
 -- Page 595, Offset 12413.
 -- 氷河によって波だつ足場。
-Footsteps wobbled by icy torrents.
+They struggle to keep their footing atop the frozen glacier.
 }
 [sha:281454b173d78fb7f73da5693ad9f6352798dbff]{
 -- Page 595, Offset 12414.
 -- 呼吸さえ死に繋がる極限状態で、氷弾と弾丸がぶつかり合う。
-Ice projectiles and bullets are colliding in this extreme situation where even breathing can lead to death,
+Under these extreme conditions,
+ where even breathing in could spell death,
+ bullets go head to head against flying shards of ice.
 }
 [sha:6e4ca612be8521acdae0b6e478f448ea8694a102]{
 -- Page 595, Offset 12415.
 -- 両者の戦いが始まってから60分。
-It's been an hour since their battle has begun.
+It's been an hour since their battle began.
 }
 [sha:32bb027e039aaf9a4d15fb7947f83bf9cbcb49fe]{
 -- Page 595, Offset 12416.
 -- その戦闘時間自体が、既に一つの奇蹟だった。
-The duration of this fight itself is already a miracle.
+That it's lasted this long is already a miracle in itself.
 }
 [sha:1cdc7a9ca3e3d73cf76f521667c95826db9f09cd]{
 -- Page 596, Offset 12417.
 -- 7.62mmの火花を散らす投石機。
-A catapult making 7.62mm sparks fly.
+// Faux-metric value spotted
+// Localized to America units because otherwise 7.62mm < 8 < 9mm and that
```

Comment:
Are you sure this is a real concern? Mind you I have not only americabrain but rural americabrain so I'm not a great source here, but 7.62 I'd imagine is pretty well known to people who play video games thanks to AK-47 ubiquity 

It doesn't really matter either way

Reply:
Well, _I_ was confused and had to go look up how bullet numbering works 
![AokoHeh](https://user-images.githubusercontent.com/2476799/184778882-89cf9dba-e127-4cd2-88e3-7484e5e4a1b0.png)

I feel like 'cal' feels weightier also? I'd probably lean towards keeping it.

Reply:
sure, not a big deal.